Gemfile

```diff
@@ -1,8 +1,9 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/butteff/is_dark_ruby_gem.git" }
 
-ruby "3.2.3"
-
 gem "rmagick"
-gem "rspec"
-gem "minitest"
\ No newline at end of file
+
+group  :development, :test do
```

Comment:
It would be cleaner to specify the exact environments in which we use these gems.